CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## v4.9.3 - 2024-01-16
```

Comment:
🐼 Are you supposed to keep an `Unreleased` header above this?

Reply:
I added this one so apparently not 🤷.

Have amended it back in though 👍.